### DISCUSSION TITLE
Question on behavior of Programs

### DISCUSSION THREAD

**Discussion by akballow** [sebr/bhyve-home-assistant#63] [Q&A]
Hello, 
Thanks in advance for looking at my question.

I have everything working on home assistant and I created a program on B-bhyve app. The program is a manual, not smart, program where it turns on zones 1 through 3 one at a time for 25 minutes. So that works and home assistant see's that it starts, but my issue is Zone 1 then Zone 2 remain on until Zone 3 finishes. This at the end makes my report look like Zone 1 was watered for 75 minutes, Zone 2 for 50 minutes, and finally Zone 3 for 25 minutes, when in reality they all were 25 minutes each. Is this expected behavior or do i need to make 3 programs for each zone to get around this issue? 

Thanks!

**Comment by sebr** [comment]
Hi @akballow - this is a known issue, please see https://github.com/sebr/bhyve-home-assistant/issues/41

I am not sure how to fix this in the HA integration at this point in time, and I don't have any devices which have multiple zones. Any thoughts or investigation you can do would be great.
